native/cocos/application/ApplicationManager.cpp

```diff
@@ -33,6 +33,12 @@ ApplicationManager *ApplicationManager::getInstance() {
     return &mgr;
 }
 
+void ApplicationManager::closeAllApplications() {
+    for (auto &app : _apps) {
```

Comment:
Use `for (auto* app : _apps)`
`auto*` for pointer rather than `auto&`

Reply:
ApplicationPtr is not a raw pointer. The code will compile with errors

Reply:
Got it.